mreg/errors.py

```diff
@@ -0,0 +1,4 @@
+from rest_framework import (exceptions, status)
+
+class ValidationError409(exceptions.APIException):
+    status_code = status.HTTP_409_CONFLICT
```

Comment:
Except that this file should probably reside under `mreg/api/` or `mreg/api/v1/` it looks good.

Reply:
That makes sense.  Moved to `mreg.api.errors`.  Thanks a lot for the feedback!  :-)